libs/coin-modules/coin-near/src/js-buildTransaction.ts

```diff
@@ -20,12 +18,7 @@ export const buildTransaction = async (
     publicKey,
   });
 
-  const currency = getCryptoCurrencyById("near");
-  const formattedAmount = formatCurrencyUnit(currency.units[0], t.amount, {
-    disableRounding: true,
-    showAllDigits: true,
-  });
-  const parsedNearAmount = nearAPI.utils.format.parseNearAmount(formattedAmount);
```

Comment:
All of that is unnecessary as a LL transaction amount is always the atomical unit. Just need to have it as a stringified BigNumber instead of a BigNumber from `bignumber.js` (while Near SDK uses `bn.js`)